packages/origin-247-claim/src/claim.module.ts

```diff
@@ -0,0 +1,46 @@
+import { Module } from '@nestjs/common';
+import { CertificateModule } from '@energyweb/origin-247-certificate';
+import { ClaimService } from './claim.service';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { MatchResultEntity } from './repositories/MatchResult/MatchResult.entity';
+import { ExcessGenerationEntity } from './repositories/ExcessGeneration/ExcessGeneration.entity';
+import { LeftoverConsumptionEntity } from './repositories/LeftoverConsumption/LeftoverConsumption.entity';
+import { MATCH_RESULT_REPOSITORY } from './repositories/MatchResult/MatchResult.repository';
+import { MatchResultPostgresRepository } from './repositories/MatchResult/MatchResultPostgres.repository';
+import { LEFTOVER_CONSUMPTION_REPOSITORY } from './repositories/LeftoverConsumption/LeftoverConsumption.repository';
+import { LeftoverConsumptionPostgresRepository } from './repositories/LeftoverConsumption/LeftoverConsumptionPostgres.repository';
+import { EXCESS_GENERATION_REPOSITORY } from './repositories/ExcessGeneration/ExcessGeneration.repository';
+import { ExcessGenerationPostgresRepository } from './repositories/ExcessGeneration/ExcessGenerationPostgress.repository';
+
+@Module({
+    imports: [
+        TypeOrmModule.forFeature([
+            MatchResultEntity,
+            LeftoverConsumptionEntity,
+            ExcessGenerationEntity
+        ]),
+        CertificateModule
+    ],
+    providers: [
+        ClaimService,
+        {
+            provide: MATCH_RESULT_REPOSITORY,
+            useClass: MatchResultPostgresRepository
```

Comment:
Where are there repositorym modules? xd

Reply:
There are no modules for those repositories, they are part of this module.

Reply:
I mean they are not in CR, I guess they were implemented already? xddd

Reply:
Yes, they were implemented in claiming-logs pr/branch